src/lib/grid-list/grid-list.ts

```diff
@@ -40,10 +41,14 @@ const MAT_FIT_MODE = 'fit';
   host: {
     'class': 'mat-grid-list',
   },
+  providers: [{
+    provide: MAT_GRID_LIST,
+    useExisting: MatGridList
```

Comment:
I don't quite follow why this is necessary- wouldn't injecting `MatGridList` do the same thing?

Reply:
It's there to avoid a circular import between the grid list and tile. The grid list has an import of the tile already.